### PR TITLE
removed property files

### PR DESCRIPTION
**Pull request by jpatton-USGS**
This PR removes the property file (and cwbproperty class) associated with neic-traveltime in favor of an optional path passed as a function argument with a default of "./models/". At this time I have done the minimum required to support what neic-locator expects, with the intent that this path will become a command line argument or service config file during the full neic-traveltime code review.

### CODE REVIEW

File: src/gov/usgs/traveltime/AuxTtRef.java [hunk 1]
```diff
@@ -83,10 +85,10 @@ public AuxTtRef(boolean readStats, boolean readEllip, boolean readTopo)
 		FileLock lock;
 		EllipDeps eDepth;
 		
-		// Set up the properties.
-		if(TauUtil.modelPath == null) {
-			TauUtil.getProperties();
+		if (modelPath != null) {
+			TauUtil.modelPath = modelPath;
```
Comment (jmfee-usgs) [inline src/gov/usgs/traveltime/AuxTtRef.java]: Can this be made non-global.

An example use case is multiple AuxTtRef's processing different model directories from the same process...

Comment (jpatton-USGS) [reply src/gov/usgs/traveltime/AuxTtRef.java]: Not easily, a number of different classes reference the tau util model path

Comment (jmfee-usgs) [reply src/gov/usgs/traveltime/AuxTtRef.java]: Same issue as https://github.com/usgs/neic-locator/pull/21#discussion_r264328716 , using global's makes it difficult to access multiple modelPaths within the same process.  Can those other things using model path receive it as a parameter or similar?

Comment (jpatton-USGS) [reply src/gov/usgs/traveltime/AuxTtRef.java]: My goal with the TT library right now is to keep it functional to support neic-locator and locator evaluation. I would prefer that we make this comment into an issue that can be addressed when I do the code review/standards pass for TT lib after the locator is in testing.

**Comment by jmfee-usgs** [comment]
https://github.com/usgs/neic-traveltime/issues/31

**Pull request merged**